Libs/DICOM/Core/ctkDICOMQuery.cpp

```diff
@@ -49,7 +51,7 @@
 #include <dcmtk/ofstd/ofstd.h>        /* for class OFStandard */
 #include <dcmtk/dcmdata/dcddirif.h>   /* for class DicomDirInterface */
 
-static ctkLogger logger ( "org.commontk.dicom.DICOMQuery" );
+dcmtk::log4cplus::Logger rootLogQuery = dcmtk::log4cplus::Logger::getRoot();
```

Comment:
Could you add a comment everywhere where now the root logger is used that explains why it is done like this?